nion/swift/ExportDialog.py

```diff
@@ -101,6 +101,26 @@ def on_writer_changed(self, widget: Declarative.UIWidget, current_index: int) ->
         writer = self.__writers[current_index]
         self.viewmodel.writer.value = writer
 
+    def on_prefix_changed(self, widget: Declarative.UIWidget, text: str) -> None:
```

Comment:
Can this be named `_handle_prefix_changed`. The variables named `on_xyz` are usually function pointers, not handlers. And the `_` prefix will denote it as "as private as can be when passing to Declarative UI". Thanks.